helm/net-exporter-chart/values.yaml

```diff
@@ -17,9 +17,11 @@ image:
   name: giantswarm/net-exporter
   tag: [[ .SHA ]]
 
-namespaceLabeler:
+kubectl:
   image:
-    name: giantswarm/namespace-labeler
+    registry: quay.io
+    name: giantswarm/docker-kubectl
+    tag: 1.16.4
```

Comment:
this a problematic, this repo has the old architect build witch is doing only SHA push, so the tag is created manually

and this will definitely once break for china as somebody will forget to create the tag in china, if there is no automatic way of the tags  i cannot approve this